client/src/document/on-github.tsx

```diff
@@ -62,6 +62,7 @@ MDN URL: https://developer.mozilla.org$PATHNAME
 * Folder: \`$FOLDER\`
 * MDN URL: https://developer.mozilla.org$PATHNAME
 * GitHub URL: $GITHUB_URL
+* Last commit: $LAST_COMMIT_URL
```

Comment:
If the line above mentions that it's a "URL", should this line do that too? I.e. `Last commit URL:`.